docker-compose.yml

```diff
@@ -484,11 +485,28 @@ services:
     environment:
       POSTGRES_DB: nextcloud
       POSTGRES_PASSWORD: postgres
+    ports:
+      - 5432:5432
     expose:
       - 5432
     volumes:
       - postgres:/var/lib/postgresql
 
+  pgadmin:
+    container_name: pgadmin_container
+    image: dpage/pgadmin4
+    environment:
+      VIRTUAL_HOST: "pgadmin${DOMAIN_SUFFIX}"
+      PGADMIN_DEFAULT_EMAIL: pgadmin4@nextcloud.local
+      PGADMIN_DEFAULT_PASSWORD: postgres
+      PGADMIN_CONFIG_SERVER_MODE: 'False'
+    volumes:
+       - pgadmin:/var/lib/pgadmin
+    ports:
+      - "${PGADMIN_PORT:-5050}:5050"
```

Comment:
No need for exposing the port if we provide a virtual host entry point through the proxy.